src/style.css

```diff
@@ -1,3 +1,61 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: black;
+}
+
+.header {
+  background-color: white;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo--img {
+  height: 40px;
+  width: 40px;
```

Comment:
Checklist - 4th point:

I know that I may be picking on, but it would be nice to create a CSS variable like `logo dimensions` for this `40px` value.

Reply:
Same thing could be done for the blue 'active' color

Reply:
Thanks for the feedback! I've made the changes to the code according to your instructions. Please let me know if everything is in order now. :)